Base/usr/share/man/man2/posix_spawn_fle_actions_init.md

```diff
@@ -0,0 +1,50 @@
+## Name
+
+`posix_spawn` - launch a new process
```

Comment:
But this is a man page for `posix_spawn_file_actions_t`, not for `posix_spawn`.

Which, while we're at it, should all be on one page, ideally.

Reply:
No, most calls to `posix_spawn` pass nullptr for both file actions and attribs. It makes sense to keep the main manpage focused for the common case.

Reply:
I don't think there's any problem with adding more information to the `posix_spawn` page — perhaps as separate sections of you don't want to clutter "description". (Also adding a bunch of examples, both simple and complex, wouldn't hurt 😉)

But I do see an issue with creating a bazillion pages (especially if you count all those symlinks) to describe various helpers of `posix_spawn`.

Reply:
I added a short example inline. I want to add a longer example in the directory with the example programs (and the mention it here once it exists).